Detections/SecurityEvent/UserAccountCreatedDeleted_10m.yaml

```diff
@@ -25,21 +25,57 @@ query: |
   let timeframe = 1d;
   let spanoftime = 10m;
   let threshold = 0;
-  SecurityEvent
+   (union isfuzzy=true
+   (SecurityEvent
   | where TimeGenerated > ago(timeframe+spanoftime)
   // A user account was created
   | where EventID == 4720
   | where AccountType =~ "User"
   | project creationTime = TimeGenerated, CreateEventID = EventID, CreateActivity = Activity, Computer, TargetUserName, UserPrincipalName, 
   AccountUsedToCreate = SubjectAccount, SIDofAccountUsedToCreate = SubjectUserSid, TargetAccount = tolower(TargetAccount), TargetSid
+  ),
+  (
+  WindowsEvent
+  | where TimeGenerated > ago(timeframe+spanoftime)
+  // A user account was created
+  | where EventID == 4720
+  | extend SubjectUserSid = tostring(EventData.SubjectUserSid)
+  | extend SubjectAccount = strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend AccountType=case(SubjectAccount endswith "$" or SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", isempty(SubjectUserSid), "", "User")
```

Comment:
Let's not go back.... but, efficiency wise, this should use **SubjectUserName** endswith "$" and lines 44 and 45 should precede 43 (after asigning SubjectUserName to a temp.